apps/testingaccessibility/studio/schemas/objects/bodyVideo.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import ReactHlsPlayer from 'react-hls-player'
+// import ReactMarkdown from 'react-markdown'
+
+export default {
```

Comment:
all videos will be in MUX, if that has any bearing here. They have a Sanity integration...

Reply:
It's clunky and I'm having issues with getting the preview to render plus if used in PortableText, only `_ref` string is getting passed down to a component which isn't supposed to be the case. I'll look into it more but for now we can use it to upload mp4s while still providing the hls url as a string.

Reply:
ya I don't think it's really intended for this purpose and we can likely just manage MUX manually